sites/default/config/views.view.search.yml

```diff
@@ -1,1331 +0,0 @@
-uuid: 5c386e62-a58e-4a80-9507-7dc6cef3cf2e
```

Comment:
This file shouldn't be deleted. Before removing the fields, make sure you are starting from a fresh revert of all the config on the site.